src/components/Results/Results.tsx

```diff
@@ -67,45 +129,55 @@ const Results: React.FC<Props> = ({ measureFile, patientFile, htmls }) => {
-            {results && (
-              <IconButton
-                onClick={() => {
-                  fileDownload(
-                    JSON.stringify(results, null, 2),
-                    measureFile.name?.includes('.json')
-                      ? `results-${measureFile.name}`
-                      : `results-${measureFile.name}.json`
-                  );
-                }}
-              >
-                <GetApp fontSize="small" />
-              </IconButton>
-            )}
-          </Grid>
-          {results && <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />}
-          {patientFile && <h2>Patient Bundle:</h2>}
-          {patientFile && (
-            <ReactJson src={patientFile} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />
-          )}
-        </Grid>
-        <Grid item xs>
-          {results &&
-            htmls &&
-            htmls.map(html => {
-              return (
-                <div key={html.groupId} className={classes.highlightedMarkup}>
-                  <h2>HTML:</h2>
-                  {parse(html.html)}
-                </div>
-              );
-            })}
-        </Grid>
-      </Grid>
+      </>
+    );
+  };
+
+  const shouldDisplayHTML = () => {
+    return htmls.length > 0;
+  };
+
+  const shouldDisplayAccordion = () => {
+    return outputType === 'gapsInCare';
+  };
+
+  const shouldDisplayTabular = () => {
+    return outputType === 'measureReports' && calculationOptions.reportType === 'individual';
+  };
+
+  const shouldDisplayTabs = () => {
+    return shouldDisplayHTML() || shouldDisplayAccordion() || shouldDisplayTabular();
+  };
+
+  return (
+    <Grid container xs={12} direction="column" spacing={2}>
+      <h2>Results:</h2>{' '}
+      {shouldDisplayTabs() ? (
+        <TabContext value={value}>
+          <AppBar position="static">
+            <Tabs
+              TabIndicatorProps={{ style: { background: 'white' } }}
+              aria-label="results tabs"
+              onChange={handleChange}
+              value={value}
+              indicatorColor="primary"
+              centered
+            >
+              <Tab style={style} label="Raw JSON" value="0" />
+              {shouldDisplayHTML() && <Tab style={style} label="Highlighted HTML" value="1" />}
+              {shouldDisplayTabular() && <Tab style={style} label="Tabular" value="2" />}
+              {shouldDisplayAccordion() && <Tab style={style} label="Accordion" value="3" />}
+            </Tabs>
+          </AppBar>
+
+          <TabPanel value="0">{displayJSONResults()}</TabPanel>
+          {shouldDisplayHTML() && <TabPanel value="1">{displayHTMLResults()}</TabPanel>}
+          {shouldDisplayTabular() && <TabPanel value="2">{displayTabularResults()}</TabPanel>}
+          {shouldDisplayAccordion() && <TabPanel value="3">{displayAccordionResults()}</TabPanel>}
+        </TabContext>
+      ) : (
+        displayJSONResults()
+      )}
```

Comment:
Now that we're using an `AppBar`, we might be able to clean up the top of the page a bit. The results/Home button look kind of out of place now with the blue app bar under them.

We might be able to just do away with the "Results" `h2` entirely now that the tabs indicate what you're viewing. On that end, I think it might make sense to always show the tabs, even when JSON is the only output. This will only generate 1 tab yes, but it makes the behavior more consistent.

Reply:
In addition, maybe we can embed the home/go back button directly into the appbar now? rather than having a separate button live above it. Thoughts?